### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by MuhammadKhalilzadeh**
Screenshot (59)

### CODE REVIEW

**Review by popnfresh234 — APPROVED** [review]
Looks good overall, some hardcoded values that could be refactored to save us headaches down the road.

No major issues I see!

File: Client/src/Components/Checkbox/Checkbox.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./Checkbox.css";
+
```
[the remark this answers is not in the document]
Comment (MuhammadKhalilzadeh) [reply Client/src/Components/Checkbox/Checkbox.jsx:3]: I texted the one for MUI, but it came to realize that it is un-convert-able to our designs and could not change value of margins and other attributes. So, I had no choice other that building out own.
This one is much more flexible by the way, compared to MUI's.

File: Client/src/Components/TextFields/Password/PasswordTextField.jsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                  {error && !showIcon && (
+                    <ErrorOutlineIcon
+                      className="text-field-icon"
+                      style={{ fill: "red", width: "16px", height: "16px" }}
+                    />
+                  )}
+                  {!error && showIcon && (
+                    <HelpOutlineIcon
+                      className="text-field-icon"
+                      style={{ width: "16px", height: "16px" }}
```
[the remark this answers is not in the document]
Comment (MuhammadKhalilzadeh) [reply Client/src/Components/TextFields/Password/PasswordTextField.jsx:73]: Oh, okay, so I'll add it to the theme as a global constant,

File: Client/src/Pages/Login/index.jsx [hunk 3]
```diff
@@ -1,10 +1,65 @@
...
+          <span className="new-account-option-span">Sign up</span>
+        </div>
+      </div>
+    </div>
+  );
+};
 
-export default Login
\ No newline at end of file
+export default Login;
```
[the remark this answers is not in the document]
Comment (MuhammadKhalilzadeh) [reply Client/src/Pages/Login/index.jsx:65]: : )))

File: Client/src/Components/Checkbox/Checkbox.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (popnfresh234) [reply Client/src/Components/Checkbox/Checkbox.jsx:3]: Sounds good, if the MUI component doesn't meet our needs then by all means :smile:

File: Client/src/Pages/Login/index.jsx [hunk 4]
```diff
@@ -1,10 +1,65 @@
...
 const Login = () => {
-    return (
-        <div>Login</div>
-    )
-}
+  return (
+    <div className="login-page">
+      <BackgroundPattern></BackgroundPattern>
+      <div className="login-form">
+        <div className="login-form-header">
```
[the remark this answers is not in the document]
Comment (MuhammadKhalilzadeh) [reply Client/src/Pages/Login/index.jsx:16]: Lot's of trouble for binding them according to our values and constants.
I'm afraid there maybe need to adjust many values and class for MUI components like these.
What do you think?

[the remark this answers is not in the document]
Comment (popnfresh234) [reply Client/src/Pages/Login/index.jsx:16]: If they don't fit well for our purposes then it's probably better to roll our own.  I haven't played around with them much at all so I will defer to those with more experience here :+1:

[the remark this answers is not in the document]
Comment (Skorpios604) [reply Client/src/Pages/Login/index.jsx:16]: I have been able to use the theme pretty consistently. I haven't ran into any issues yet. But if there are issues then we could use the index.css for styling like @veyselboybay suggested.

File: Client/src/Components/Checkbox/Checkbox.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (MuhammadKhalilzadeh) [reply Client/src/Components/Checkbox/Checkbox.jsx:3]: Yeah, I do check the documentation, but the problem is that even those classes need to have some of their values adjusted according to developers needs.

**Pull request merged**